train.py

```diff
@@ -311,20 +313,20 @@ def fit(self, epochs):
                 self.val_losses.append(val_mean)
                 val_losses.append(val_mean)
 
-                if ((epoch + 1) % print_interval) == 0:
-                    print(f"Epoch {epoch + 1} Time taken {round(time.time() - start, 2)} seconds "
-                          f"Train Loss {train_losses[-1]} Val Loss {val_losses[-1]}")
-                    # print("Checkpointing...")
-                    # self.save()
-                    # print("Done")
-                    start = time.time()
+                print(f"Epoch {epoch } Time taken {round(time.time() - start, 2)} seconds "
+                    f"Train Loss {train_losses[-1]} Val Loss {val_losses[-1]}")
+                # print("Checkpointing...")
+                # self.save()
+                # print("Done")
```

Comment:
These are comments that I forgot to delete, please could you delete them in your branch?

Reply:
Will do

Reply:
Removed